### PR TITLE
Add browserify peerDependency and test all major versions

### PR DESCRIPTION
**Pull request by bendrucker**
test-peer-range iterates across major versions and helps defend against regressions against old versions or breaking changes to browserify that affect deamdify

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,11 +34,16 @@
...
     "mocha": "^2.2.4",
-    "chai": "^2.2.0"
+    "test-peer-range": "^1.0.1"
+  },
+  "peerDependencies": {
+    "browserify": ">= 2"
   },
   "scripts": {
-    "test": "mocha --reporter spec --require test/bootstrap/node test/*.test.js"
+    "test": "test-peer-range browserify"
```
Comment (tbranyen) [inline package.json]: Ah, missing comma end of this line.

**Comment by bendrucker** [comment]
:fearful: Fixed!

**Comment by tbranyen** [comment]
Okay so we `npm run test-main` now to test instead of `npm test`.  Can you update the README code block in this PR as well (to show this new command)? If not, I'll merge and update.

**Comment by bendrucker** [comment]
Sort of. test-peer-range can use a custom script and so we could reverse this and do something like test-all for the full run and then just change the `script` for Travis. I'll update the README to show both test and test-main but if you wanna just do this for Travis I can amend.

**Comment by tbranyen** [comment]
Looks good, thank you!

File: README.md [hunk 2]
```diff
@@ -58,8 +58,12 @@ package as it builds a bundle.
...
 ## Tests
 
-    $ npm install
-    $ npm test
+```sh
+$ npm install
+# fast run
+$ npm run test-main
+# test all browserify major versions
+$ npm test
```
Comment (tbranyen) [inline README.md]: Noticed the missing ``` too late, added in fix here: 67c571a8a03f705a31e59fbf954f10ec477cda47

**Pull request merged**